docs/intro.rst

```diff
@@ -3,15 +3,20 @@
 Introduction
 ============
 
+*The relational model is still the most generally applicable and effective method of representing data. The concept of SQL as a declarative, storage-neutral query language is powerful and versatile. We don’t need to abandon either. What we do need is a “better SQL”: a query language that affords more power to its users, but that is also simpler and more consistent.*
```

Comment:
Maybe we can express the same idea in simple English (and not quoting the blog post)? I'm also not sure that saying "better SQL" is a good idea -- it might trigger some of its fans to rage-close the page. But I love the idea of briefly explaining what EdgeDB is for to the user in the first paragraph.

Reply:
True, don't want any rage quits. I'll simplify the English tomorrow (just about to shut off for the night here) based on this idea rewriting the "better SQL" part to make it more of a compliment:

*The relational model is still the most generally applicable and effective method of representing data, and the concept of SQL as a declarative, storage-neutral query language is powerful and versatile. We don’t need to abandon either. This is why we build on what works to make EdgeDB what it is: a query language that affords more power to its users, but that is also simpler and more consistent.*